web-common/src/components/overlay/Overlay.svelte

```diff
@@ -9,10 +9,8 @@
   // We'll need a better solution than this!
   function captureKeydown(event) {
     // capture all events
-    if (event.code === "Tab" || event.code === "Space") {
-      document.activeElement.blur();
-      event.preventDefault();
-    }
+    document.activeElement.blur();
```

Comment:
does this block anything on top of the overlay that might take input?

Reply:
Didn't get you. Isn't the overlay on top of everything else?

Reply:
oh actually, nevermind, there are two kinds of overlays in our repository ~ this one, and the basis for a Modal. These function differently and will need to be handled separately.